src/monattack.cpp

```diff
@@ -4345,7 +4345,7 @@ bool mattack::riotbot(monster *z)
                 foe->i_add(handcuffs);
             } else {
                 handcuffs.item_tags.insert("NO_UNWIELD");
-                foe->wield(&(foe->i_add(handcuffs)));
+                foe->wield( foe->i_add( handcuffs ) );
```

Comment:
> How does this part work with atomic wield?

It's still atomic, it's just that the caller doesn't checker the result and therefore isn't

> I assume on failure, the cuffs will end up in the inventory
> Later on it probably should handle failure to cuff

Both yes

> Couldn't it be simplified to just foe->wield( handcuffs )

Agreed that's a trivial change although it's still broken after this (and always was); before this PR you could still press `N` at the prompt whereas now you can escape from the menu.

A more complete fix would be when we implement automatic actions to specify `DROP` forcing the player to automatically drop any existing weapon.